graylog_hook_test.go

```diff
@@ -283,12 +285,26 @@ func TestStackTracer(t *testing.T) {
 		t.Error("Stack Trace is not a string")
 	}
 	stacktraceRE := regexp.MustCompile(`^
-.+/logrus-graylog-hook(%2ev2)?.TestStackTracer
+(.+)?logrus-graylog-hook(%2ev2)?.TestStackTracer
 	(/|[A-Z]:/).+/logrus-graylog-hook(.v2)?/graylog_hook_test.go:\d+
 testing.tRunner
 	(/|[A-Z]:/).*/testing.go:\d+
 runtime.*
 	(/|[A-Z]:/).*/runtime/.*:\d+$`)
+
+	// Change the regex when running test on devel version since stack trace on devel is unpredictable
+	if strings.Contains(runtime.Version(), "devel") {
```

Comment:
This was the cleanest way I can think to make the tip pass. I noticed that except from the new lines
that are matched with 

```
runtime.skipPleaseUseCallersFrames
	(/|[A-Z]:/).*/asm.s:\d+
```
this line 

```
(/|[A-Z]:/).+/logrus-graylog-hook(.v2)?/graylog_hook_test.go:\d+
```

should also change to this

```
(/|[A-Z]:/).*/errors.go:\d+
```

Since this test is so fragile on tip, why not to skip it completely on devel version and keep it only for stable go versions?
 
I will be glad to make this change. It will make things easier :)